config/environments/test.rb

```diff
@@ -16,26 +16,26 @@
   # Configure public file server for tests with Cache-Control for performance.
   config.public_file_server.enabled = true
   config.public_file_server.headers = {
-    "Cache-Control" => "public, max-age=3600",
+    "Cache-Control" => "public, max-age=#{1.hour.to_i}",
   }
 
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
+  config.cache_store = :null_store
 
   # Raise exceptions instead of rendering exception templates.
   config.action_dispatch.show_exceptions = false
 
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
-  config.active_support.test_order = :random
```

Comment:
Looks as if the PR takes this line out but was there a reason this was added in the first place?

Reply:
That defaults to random in any case, so seems unnecessary?